snapshot-tests/src/test/java/io/eventuate/examples/tram/ordersandcustomers/snapshottests/SnapshotTest.java

```diff
@@ -0,0 +1,131 @@
+package io.eventuate.examples.tram.ordersandcustomers.snapshottests;
+
+import io.eventuate.examples.tram.ordersandcustomers.CustomerTextView;
+import io.eventuate.examples.tram.ordersandcustomers.OrderTextView;
+import io.eventuate.examples.tram.ordersandcustomers.commondomain.Money;
+import io.eventuate.examples.tram.ordersandcustomers.customers.webapi.CreateCustomerRequest;
+import io.eventuate.examples.tram.ordersandcustomers.customers.webapi.CreateCustomerResponse;
+import io.eventuate.examples.tram.ordersandcustomers.orders.webapi.CreateOrderRequest;
+import io.eventuate.examples.tram.ordersandcustomers.orders.webapi.CreateOrderResponse;
+import io.eventuate.tram.viewsupport.rebuild.TopicPartitionOffset;
+import io.eventuate.util.test.async.Eventually;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = SnapshotTestConfiguration.class, webEnvironment = SpringBootTest.WebEnvironment.NONE)
+public class SnapshotTest {
+
+  @Value("localhost")
```

Comment:
`@Value(constant) ` can be just turned into a field assignment

Reply:
ok